docs/fips.md

```diff
@@ -0,0 +1,7 @@
+# FIPS compliance
+
+FIPS (Federal Information Processing Standard) is the US government computer security standard for cryptography modules that include both hardware and software components. Percona Server for MySQL supports FIPS certified module for OpenSSL, enabling US organizations to introduce FIPS-compliant encryption and thus meet the requirements towards data security.  
+
+Percona Server for MySQL implements the same level of FIPS support as MySQL. You can also receive this functionality by [building Percona Server for MySQL from source code](compile-percona-server.md).
+
+See [Configure MySQL for FIPS](https://dev.mysql.com/doc/refman/8.0/en/fips-mode.html) in MySQL documentation for configuration guidelines. 
```

Comment:
Ask what the variables are and the parameters.

Reply:
Updated